recipes/grass/meta.yaml

```
@@ -0,0 +1,144 @@
+{% set name = "grass" %}
+{% set version = "8.4.1" %}
+{% set sha256 = "611e82dd919cd0e5c1a511e7efbc5c769e22ac6c613c3c4a2633c5d4b07c7e31" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OSGeo/grass/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or osx or py<310]
+  ignore_run_exports:
+    - gdal
+    - libtiff
+    - expat
+    - numpy
+    - zlib
+    - pixman
+    - ncurses
+    - python
+    - sqlite
+    - xorg-xextproto
+    - libmagic
+    - jsoncpp
+    - pdal
+    - libjpeg-turbo
+    - freetype
+    - libexpat
+  # GRASS uses Autotools configure/make; build.sh handles the steps
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - make
+    - pkg-config
+    - bison
+    - flex
+    - gettext
+  host:
+    - python
+    - pip
+    - gdal
+    - geos
+    - proj
+    - proj-data
+    - sqlite
+    - readline
+    - ncurses
+    - cairo
+    - freetype
+    - fontconfig
+    - expat
+    - pixman
+    - xorg-libx11
+    - xorg-xproto
+    - xorg-kbproto
+    - xorg-xextproto
+    - xorg-renderproto
+    - libpng
+    - libjpeg-turbo
+    - libtiff
+    - zlib
+    - zstd
+    - bzip2
+    - libpq
+    - jsoncpp
+    - libmagic
+    - fftw
+    - blis
+    - liblapack
+    - libnetcdf
+    - pdal
+    - libiconv
+    - setuptools
+  run:
+    - python
+    - libiconv
+    - gdal
+    - libgdal-core
+    - geos
+    - proj
+    - proj-data
+    - sqlite
+    - readline
+    - ncurses
+    - cairo
+    - freetype
+    - fontconfig
+    - expat
+    - libpng
+    - libjpeg-turbo
+    - libtiff
+    - zlib
+    - zstd
+    - bzip2
+    - libpq
+    - jsoncpp
+    - libmagic
+    - fftw
+    - blis
+    - liblapack
+    - libnetcdf
+    - pdal
+    - libpdal-core
+    - _openmp_mutex
+    # Python modules typically used by GRASS python tools
+    - numpy
+    - pillow
+    - matplotlib-base
+    - python-dateutil
+    - ply
+    - psycopg2
+    - python-magic
+
+test:
+  commands:
+    - grass --version
+    - grass --tmp-project EPSG:4326 --exec g.version -rge
+  requires:
+    - pip
+
+about:
+  home: https://grass.osgeo.org/
+  license: GPL-2.0-or-later
+  license_file: COPYING
+  summary: GRASS GIS for geospatial data management and analysis
+  description: |
+    GRASS (Geographic Resources Analysis Support System) is a free and 
+    open source Geographic Information System (GIS) used for geospatial 
+    data management and analysis, image processing, graphics/map production, 
+    spatial modeling, and visualization. 
+  dev_url: https://github.com/OSGeo/grass
+  doc_url: https://grass.osgeo.org/grass-stable/manuals/index.html
```

Some suggestions for summary and description:

```suggestion
  license_file: COPYING
  summary: GRASS computational engine for raster, vector, and geospatial processing. 
  description: |
    GRASS is a geospatial engine that enables spatial data analysis,
    visualization, and modeling. It provides researchers and practitioners
    with science-backed tools to understand and manage landscapes,
    natural resources, and human environments.
  dev_url: https://github.com/OSGeo/grass
  doc_url: https://grass.osgeo.org/grass-stable/manuals/
```